Task_1/Square.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_1
+{
+    public class Square : Figure
+    {
+        protected int Side { get; set; }
```

Comment:
No need to use protected here since you don't have inheritors and uses this inside this class only.
So should be private.